services/121-service/src/scripts/seed-program-pilot-nl.ts

```diff
@@ -0,0 +1,67 @@
+import { Injectable } from '@nestjs/common';
+import { InterfaceScript } from './scripts.module';
+import { Connection } from 'typeorm';
+
+import { SeedHelper } from './seed-helper';
+import { SeedInit } from './seed-init';
+
+import { CountryEntity } from '../programs/country/country.entity';
+
+import fspBank from '../../examples/fsp-bank.json';
+import fspMobileMoney from '../../examples/fsp-mobile-money.json';
+import { ProtectionServiceProviderEntity } from '../programs/program/protection-service-provider.entity';
+
+import programPilotNL from '../../examples/program-pilot-nl.json';
+import { USERCONFIG } from '../secrets';
+import { UserRole } from '../user-role.enum';
+
+@Injectable()
+export class SeedPilotNLProgram implements InterfaceScript {
+  public constructor(private connection: Connection) {}
+
+  private readonly seedHelper = new SeedHelper(this.connection);
+
+  public async run(): Promise<void> {
+    const seedInit = await new SeedInit(this.connection);
+    await seedInit.run();
+
+    await this.seedHelper.addUser({
+      role: UserRole.ProjectOfficer,
+      email: USERCONFIG.emailProjectOfficer,
+      countryId: USERCONFIG.countryId,
+      password: USERCONFIG.passwordProjectOfficer,
+    });
+
+    await this.seedHelper.addUser({
+      role: UserRole.ProgramManager,
+      email: USERCONFIG.emailProgramManager,
+      countryId: USERCONFIG.countryId,
+      password: USERCONFIG.passwordProgramManager,
+    });
+
+    // ***** CREATE COUNTRIES *****
+    const countryRepository = this.connection.getRepository(CountryEntity);
+    await countryRepository.save([{ country: 'Nederland' }]);
+
+    // ***** CREATE FINANCIAL SERVICE PROVIDERS *****
+    await this.seedHelper.addFsp(fspBank);
+    await this.seedHelper.addFsp(fspMobileMoney);
+
+    // ***** CREATE PROTECTION SERVICE PROVIDERS *****
```

Comment:
Is it technically required for these providers to be defined?
If not, we could remove it from this script.

Reply:
@elwinschmitz I'm not sure if it's technically required, but didn't want to take the risk. And it's good for the NL-pilot-team to see all the options in the interfaces at least with (such) placeholders. Such that they know it can be replaced by a real value.